fsthttp/request.go

```diff
@@ -457,23 +751,46 @@ func (req *Request) copyBody(errc chan<- error) {
 }
 
 func (req *Request) constructABIRequest() error {
-	if req.abi.req != nil || req.abi.body != nil {
-		return fmt.Errorf("request already constructed")
-	}
+	/*
+		if req.abi.req != nil || req.abi.body != nil {
+			return nil // fmt.Errorf("request already constructed")
```

Comment:
I think eliminating this is mostly OK, though it leaves open the possibility of having two calls separately construct `req` and `body`. That seems unlikely, especially since the caller protects from this.

Reply:
858e4dd